packages/frontend/src/index.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { NextUIProvider } from '@nextui-org/react';
+import { mainTheme } from './theme';
 
 import './services/firebase';
 import './styles/index.css';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <NextUIProvider theme={mainTheme}>
```

Comment:
Any chance we could move this inside the `<App />`  component (surrounding the BrowserRouter)?

Reply:
done